### PR TITLE
[Feature] Add temp_scaled_logprobs and top_p_normalized_logprobs parameters for logits and logprobs post processing

### PR DESCRIPTION
**Pull request by ckl117**
This PR introduces two new parameters to enhance log-probability sampling flexibility:

**temp_scaled_logprobs**

1. Applies temperature scaling directly to log-probabilities.
3. Helps in controlling randomness during decoding without affecting other probability adjustments.

**top_p_normalized_logprobs**

1. Normalizes log-probabilities after nucleus (top-p) filtering.
2. Ensures probability mass consistency and stable sampling behavior.

Service layer functional support [PR#3523](https://github.com/PaddlePaddle/FastDeploy/pull/3523)

```
# clinet send req
response = client.chat.completions.create(
    model="default",
    messages=[
        {"role": "user", "content": "Hello, who are you"},
    ],
    max_tokens=7,
    temperature=0.7,
    top_p=0.8,
    logprobs=True,
    top_logprobs=0,
    extra_body={
        "temp_scaled_logprobs": True,       # default False
        "top_p_normalized_logprobs": True,  # default False
    },
    stream=True,
)
```

### CODE REVIEW

**Comment by Jiang-Jia-Jun** [comment]
需要更新一下docs/serving/README.md中对于参数的说明 @ckl117

File: fastdeploy/input/ernie_processor.py [hunk 1]
```diff
@@ -479,5 +479,8 @@ def update_stop_seq(self, stop_sequences):
         return stop_seqs, stop_seqs_len
 
     def process_logprob_response(self, token_ids, **kwargs):
-        full_text = self.tokenizer.decode(token_ids, **kwargs)
+        try:
+            full_text = self.tokenizer.decode(token_ids, **kwargs)
+        except Exception:
+            full_text = "Invalid Token"
```
Comment (Jiang-Jia-Jun) [inline fastdeploy/input/ernie_processor.py]: 此处无法区别full_text是正常解码产生的文本，还是异常解码产生的文本

**Review by Jiang-Jia-Jun — CHANGES_REQUESTED** [review]

**Review by Jiang-Jia-Jun — APPROVED** [review]
